sunpy/net/vso/attrs.py

```diff
@@ -353,11 +355,11 @@ def _(attr, results):
         if
         it.time.end is not None
         and
-        attr.min <= _attrs.Time.strptime(it.time.end, _TIMEFORMAT)
+        attr.min <= parse_time(it.time.end)
```

Comment:
What was the use of `TIMEFORMAT`?

Reply:
If it's not used anywhere else, remove it.

Reply:
`_TIMEFORMAT` is being used in the converter in the same file.